Keas.Mvc/ClientApp/package.json

```diff
@@ -95,6 +95,6 @@
   },
   "engines": {
```

Comment:
Changes in npm and node usually go together. Should probably update the npm version here to whatever comes with node 20.0.0.